IDEAS/Buildings/Components/Shading/HorizontalFins.mo

```diff
@@ -0,0 +1,39 @@
+within IDEAS.Buildings.Components.Shading;
+model HorizontalFins "horizontal fins shading"
+  extends IDEAS.Buildings.Components.Shading.Interfaces.PartialShading(
+                                                             final controlled=false);
+
+  parameter Modelica.SIunits.Angle beta= Modelica.Constants.pi/6 "inclination angle of shading fins";
+  parameter Modelica.SIunits.Length l=0.170 "distance between shading fins, in meters";
+  parameter Modelica.SIunits.Length D=0.175 "size of the fins, in meters";
+  parameter Modelica.SIunits.Length w=0.032 "width of the fins, in meters";
+
+initial equation
+  assert(beta > 0 and beta < 5*Modelica.Constants.pi/12, "beta between feasible values");
```

Comment:
It is not justified and it is a random value. Maximum should be 90 degrees, but this is only possible to achieve if D < l, otherwise the fins will touch at some point. Yet to be calculated